### ISSUE TITLE
npm i pixi.js@^6.5.8 pixi-spine@^3.0.11  bugs

### ISSUE THREAD

**Issue by aram88888** [pixijs/spine#481]
After installation,  atlas and png files aren't loading.
When installing exact versions, loader worked, but we found interaction bugs.
Please find the installation log attached, where peer dependency warnings are shown.
Few days ago everything worked.
[New Text Document.txt](https://github.com/pixijs/spine/files/10499365/New.Text.Document.txt)

**Comment by miltoncandelero** [comment]
what is your npm version? mine works without any warning 🤔 
![image](https://user-images.githubusercontent.com/15253010/214573241-af755923-0eec-4d66-9f0f-77fc7c69bdd6.png)

**Comment by aram88888** [comment]
8.1.3
Please use single command, instead of installing separately.

**Comment by aram88888** [comment]
Real situation is the following. We have many projects built with
pixi-spine. I just tried to reproduce problems only with your product. Our
usual package.json looks like this.

"dependencies": {
  ***@***.***/sound": "^4.0.5",
  "pixi-scrollbox": "^2.3.1",
  "pixi-viewport": "^4.33.0",
  "pixi-spine": "^3.0.11",
  "pixi.js": "^6.5.8"
},

Everything worked fine, until today we tried to rebuild one of our projects.
we met package installation errors. I found out that, errors are connected
with pixi/sound, pixi-vieport, (connected with @pixi versions 6.5.8  and
6.5.9).  pixi-spine logs only peer dependency warrnings. When installing
exact versions, errors are gone (pixi-vieport requires to
add @***@***.*** in dependency list). Then we met run time errors
connected with pixi-spine.
If we install exact versions except for pixi.js (pixi.js installed with
^6.5.8)  Spine files except for json were not loaded.
if we install exact version also for pixi.js we have interaction error
connected with spines. Same as here
https://github.com/pixijs/pixijs/issues/7750

**Comment by miltoncandelero** [comment]
This might be related:
https://github.com/pixijs/pixijs/issues/9108

**Comment by aram88888** [comment]
Yes. That is mine too. They told it seams some packages are missing in
6.5.9. I am waiting for their final answer.
